pandas/tests/groupby/test_groupby.py

```diff
@@ -4314,6 +4314,16 @@ def test_cummin_cummax(self):
         expected = pd.Series([1, 2, 1], name='b')
         tm.assert_series_equal(result, expected)
 
+    def test_numeric_coercion(self):
+        # GH 14423
```

Comment:
add a 1-liner explaining this (maybe also make the test name a bit more descriptive)